requirements.txt

```diff
@@ -8,6 +8,8 @@ configparser==3.7.4
 cornice==3.5.1
 cryptography==2.6.1
 enum34==1.1.6
+gevent==1.4.0
+gunicorn==19.10.0
```

Comment:
(To clarify, `make install` does not install from this file, but `make tests` does because it's referenced in `dev-requirements.txt`).

Reply:
Ah! cool. I'll update make to include them. I may also update setup.py to pull the requirements from that file since it's good to bottleneck that stuff.